chapters/introduction.tex

```diff
@@ -1,27 +1,36 @@
 \begin{fullwidth}
 Welcome to Data for Development Impact.
-This book is intended to serve as a resource guide
-for people who collect or use data for development research.
-In particular, the book is intended to guide the reader
-through the process of research using primary survey data,
-from research design to fieldwork to data management to analysis.
-This book will not teach you econometrics or epidemiology or agribusiness.
-This book will not teach you how to design an impact evaluation.
-This book will not teach you how to do data analysis, or how to code.
-There are lots of really good resources out there for all of these things,
-and they are much better than what we would be able to squeeze into this book.
-
-What this book will teach you is how to think about quantitative data,
-keeping in mind that you are not going to be the only person
-collecting it, using it, or looking back on it.
-We hope to provide you two key tools by the time you finish this book.
-First, we want you to form a mental model of data collection as a ``social process'',
+This book is intended to teach all users of development data 
+how to handle data effectively, efficiently, and ethically. 
+An empirical revolution has changed the face of research economics rapidly over the last decade. 
+%had to remove cite {\cite{angrist2017economic}} because of full page width
+Economics graduate students of the 2000s expected to work with primarily ``clean'' data from secondhand sources. 
```

Comment:
I am not sure I understand what we are saying here

Reply:
it was just a comment in the latex file that i had to take out the citation because it doesn't work with the full width. we don't necessarily need to cite that sentence (fairly general), but we may want to work it back in at another point in this chapter.

Reply:
I meant "_Economics graduate students of the 2000s expected to work with primarily ``clean'' data from secondhand sources_"

Reply:
Ah, ok. The point was that now everyone is working with raw data and as recently as a decade ago that was uncommon. I'll think about how to revise so this is clearer.

Reply:
Can we say something like  "_Recent economics graduates working as RAs today, were trained by professors who did their PhD before the digital data revolution that made it more common to work with raw primary data_". I think "2000s" makes it sound so specific that it needs an explanation. But everyone know that there has been a revolution in data, and I think our point is that there is a lag in people's skills.

Reply:
If we go along the lines that I suggested we could make it less critical against schools and professors by saying that "_the math and the economic theory is the same but the data work is completely different_".

Reply:
I just deleted this line. i don't think it's needed, given the points made in the sentences just before and after.